clients/src/main/java/org/apache/kafka/clients/consumer/internals/PartitionAssignor.java

```diff
@@ -19,6 +19,7 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
+import java.util.Collection;
```

Comment:
Checkstyle doesn't like this import since it's only used in the javadoc. You might need to use the full name in the javadoc instead.